spswarehouse/powerschool/powerschool.py

```diff
@@ -631,7 +633,62 @@ def upload_quick_import(self, filename, final_value, table_name):
         self.helper_wait_for_element_containing_specific_text(final_value, 60)
         logging.info('Final student found. Upload file finished processing.')
         
-    
+    def upload_data_import_manager(self, file_path, table_name, num_rows_in_file, max_processing_wait_time_in_seconds = 60, override_existing_record = False):
```

Comment:
This needs some documentation.

E.g., The default field delimiter in the Quick Import is tabs, which is not standard practice for most people.

At some point, should also wrap this in a helper function that counts the number of rows. Take a look at the quick import functions around line 550 to see what I mean.

Reply:
Added documentation for this function and noted the need for a helper function in a TODO.